advocacy_docs/edb-postgres-ai/migration-etl/data-migration-service/index.mdx

```diff
@@ -0,0 +1,28 @@
+---
+title: EDB Data Migration Service
+indexCards: simple
+deepToC: true
+directoryDefaults:
+  description: "EDB Data Migration Service is a PG AI integrated migration solution that enables secure, fault-tolerant, and performant migrations to EDB Postgres AI Cloud Service."
+  product: "data migration service"
+  iconName: EdbTransporter
+navigation:
+  - "#Concepts"
+  - terminology
+  - "#Planning"
+  - supported_versions
+  - limitations
+  - "#Get started"
+  - getting_started
+  - "#Reference"
+  - known_issues
+
+---
+
+**EDB Postgres® AI migrations powered by EDB Data Migration Service**
+
+EDB Data Migration Service (DMS) offers a secure and fault-tolerant way to migrate database data to the EDB Postgres AI platform. Using change data capture or CDC and event streaming, source database row changes are replicated to the migration destination. You can select a subset of your schemas' tables to migrate including support for schema, table, and column name remapping.
+
```

Comment:
In addition to adding information about the public preview nature of the new service and it's availability only on AWS BAH to the compatibility page, @gvasquezvargas suggested that it might be good to add a note here as well. I think that is a good idea since users may come to this page first.

With that in mind, I suggest adding a new paragraph here with this text:

> The EDB Data Migration Service is currently a public preview capability that is available for use in the EDB Hosted EDB Postgres AI Cloud Service deployment on AWS.

Reply:
done.